types/Config.js

```diff
@@ -21,6 +21,34 @@ export type HasteConfig = {|
 
 export type ConfigGlobals = Object;
 
+export type GlobalConfig = {
+  bail: boolean,
+  collectCoverage: boolean,
+  collectCoverageFrom: Array<Glob>,
+  collectCoverageOnlyFrom: {[key: string]: boolean},
+  coverageDirectory: string,
+  coveragePathIgnorePatterns: Array<string>,
+  coverageReporters: Array<string>,
+  coverageThreshold: {global: {[key: string]: number}},
```

Comment:
I think coverage itself should be global but the coveragePathIgnorePatterns probably needs to stay in the local config. I'll change that later.

cc @dmitriiabramov what do you think?